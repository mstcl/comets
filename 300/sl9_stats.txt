Nominal rubble pile parameters (all units MKS):
(a,b,c)=(960.375,960.375,960.375) rho_b=300 N=500 r=100 rho_p=531.464
bulk mass=1.11309e+12 particle mass=2.22619e+09 color=3
Estimated packing efficiency: e_N=0.564479 e_R/r=0.559282
Adjusted parameters:
(a,b,c)=(830.768,900,940.301) N=473 rho_p=531.464
bulk mass=1.05299e+12 bulk density=357.557
Actual packing efficiency = 0.672778
Additional characteristics:
Confining radius = 956.349 (surface gravity = 7.68422e-05)
Radius of equivalent sphere = 889.194 (surface gravity = 8.88873e-05)
Escape speed from surface of particle = 0.0545131
Dynamical time for particle = 5309.56
Escape speed from surface of rubble pile = 0.383374
Surface speed at equator due to rotation = 0
Dynamical time for rubble pile = 6473.25
