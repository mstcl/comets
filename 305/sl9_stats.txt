Nominal rubble pile parameters (all units MKS):
(a,b,c)=(960.375,960.375,960.375) rho_b=305 N=500 r=100 rho_p=540.321
bulk mass=1.13165e+12 particle mass=2.26329e+09 color=3
Estimated packing efficiency: e_N=0.564479 e_R/r=0.559282
Adjusted parameters:
(a,b,c)=(830.768,900,940.301) N=473 rho_p=540.321
bulk mass=1.07054e+12 bulk density=363.516
Actual packing efficiency = 0.672778
Additional characteristics:
Confining radius = 956.349 (surface gravity = 7.81229e-05)
Radius of equivalent sphere = 889.194 (surface gravity = 9.03687e-05)
Escape speed from surface of particle = 0.0549655
Dynamical time for particle = 5265.86
Escape speed from surface of rubble pile = 0.386556
Surface speed at equator due to rotation = 0
Dynamical time for rubble pile = 6419.98
