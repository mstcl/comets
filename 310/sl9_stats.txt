Nominal rubble pile parameters (all units MKS):
(a,b,c)=(960.375,960.375,960.375) rho_b=310 N=500 r=100 rho_p=549.179
bulk mass=1.1502e+12 particle mass=2.3004e+09 color=3
Estimated packing efficiency: e_N=0.564479 e_R/r=0.559282
Adjusted parameters:
(a,b,c)=(830.768,900,940.301) N=473 rho_p=549.179
bulk mass=1.08809e+12 bulk density=369.476
Actual packing efficiency = 0.672778
Additional characteristics:
Confining radius = 956.349 (surface gravity = 7.94036e-05)
Radius of equivalent sphere = 889.194 (surface gravity = 9.18502e-05)
Escape speed from surface of particle = 0.0554142
Dynamical time for particle = 5223.22
Escape speed from surface of rubble pile = 0.389712
Surface speed at equator due to rotation = 0
Dynamical time for rubble pile = 6367.99
