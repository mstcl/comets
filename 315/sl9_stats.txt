Nominal rubble pile parameters (all units MKS):
(a,b,c)=(960.375,960.375,960.375) rho_b=315 N=500 r=100 rho_p=558.037
bulk mass=1.16875e+12 particle mass=2.3375e+09 color=3
Estimated packing efficiency: e_N=0.564479 e_R/r=0.559282
Adjusted parameters:
(a,b,c)=(830.768,900,940.301) N=473 rho_p=558.037
bulk mass=1.10564e+12 bulk density=375.435
Actual packing efficiency = 0.672778
Additional characteristics:
Confining radius = 956.349 (surface gravity = 8.06843e-05)
Radius of equivalent sphere = 889.194 (surface gravity = 9.33316e-05)
Escape speed from surface of particle = 0.0558593
Dynamical time for particle = 5181.6
Escape speed from surface of rubble pile = 0.392842
Surface speed at equator due to rotation = 0
Dynamical time for rubble pile = 6317.25
