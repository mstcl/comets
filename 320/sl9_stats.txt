Nominal rubble pile parameters (all units MKS):
(a,b,c)=(960.375,960.375,960.375) rho_b=320 N=500 r=100 rho_p=566.894
bulk mass=1.1873e+12 particle mass=2.3746e+09 color=3
Estimated packing efficiency: e_N=0.564479 e_R/r=0.559282
Adjusted parameters:
(a,b,c)=(830.768,900,940.301) N=473 rho_p=566.894
bulk mass=1.12319e+12 bulk density=381.394
Actual packing efficiency = 0.672778
Additional characteristics:
Confining radius = 956.349 (surface gravity = 8.1965e-05)
Radius of equivalent sphere = 889.194 (surface gravity = 9.48131e-05)
Escape speed from surface of particle = 0.0563009
Dynamical time for particle = 5140.96
Escape speed from surface of rubble pile = 0.395947
Surface speed at equator due to rotation = 0
Dynamical time for rubble pile = 6267.7
