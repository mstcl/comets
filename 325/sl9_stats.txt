Nominal rubble pile parameters (all units MKS):
(a,b,c)=(960.375,960.375,960.375) rho_b=325 N=500 r=100 rho_p=575.752
bulk mass=1.20585e+12 particle mass=2.41171e+09 color=3
Estimated packing efficiency: e_N=0.564479 e_R/r=0.559282
Adjusted parameters:
(a,b,c)=(830.768,900,940.301) N=473 rho_p=575.752
bulk mass=1.14074e+12 bulk density=387.354
Actual packing efficiency = 0.672778
Additional characteristics:
Confining radius = 956.349 (surface gravity = 8.32457e-05)
Radius of equivalent sphere = 889.194 (surface gravity = 9.62945e-05)
Escape speed from surface of particle = 0.056739
Dynamical time for particle = 5101.26
Escape speed from surface of rubble pile = 0.399029
Surface speed at equator due to rotation = 0
Dynamical time for rubble pile = 6219.3
