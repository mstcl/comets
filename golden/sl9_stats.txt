Nominal rubble pile parameters (all units MKS):
(a,b,c)=(478,478,478) rho_b=550 N=500 r=100 rho_p=974.35
bulk mass=2.04067e+12 particle mass=4.08135e+09 color=3
Estimated packing efficiency: e_N=0.564479 e_R/r=0.443931
Adjusted parameters:
(a,b,c)=(397.689,394.643,420.102) N=35 rho_p=974.35
bulk mass=1.42847e+11 bulk density=517.226
Actual packing efficiency = 0.530842
Additional characteristics:
Confining radius = 446.41 (surface gravity = 4.78424e-05)
Radius of equivalent sphere = 403.987 (surface gravity = 5.84178e-05)
Escape speed from surface of particle = 0.073811
Dynamical time for particle = 3921.37
Escape speed from surface of rubble pile = 0.206675
Surface speed at equator due to rotation = 0
Dynamical time for rubble pile = 5382.14
